app/forms/coronavirus_find_support/able_to_go_out_form.rb

```diff
@@ -2,5 +2,26 @@ module CoronavirusFindSupport
   class AbleToGoOutForm < Form
     answer_flow :coronavirus_find_support
     answer_node :able_to_go_out
+
+    validates :able_to_go_out, presence: { message: t("errors.blank") }
+
+    def options
+      %i[
+        yes
+        no_i_have_coronavirus
+        no_i_have_a_medical_condition
+        no_i_have_a_disability
+        no_for_another_reason
+      ]
+    end
+
+    def prepare_session
+      session[flow_name] ||= {}
+      session[flow_name]["getting_food"] ||= {}
+    end
+
+    def update_session
+      session[flow_name]["getting_food"][node_name] = data_to_be_stored_in_session
+    end
```

Comment:
_For the future perhaps:_

It would good to tidy the group name definition. Something like on line 5 (with update to `Flow` to accommodate):
```ruby
answer_group :getting_food
```
Then in the parent `Flow` class we could update `prepare_session` and `update_session` to:
```ruby
def prepare_session
  session[flow_name] ||= {}
  session[flow_name][group_name] ||= {} if group_name.present?
end

def update_session
  if group_name.present?
    session[flow_name][group_name][node_name] = data_to_be_stored_in_session
  else
    session[flow_name][node_name] = data_to_be_stored_in_session
  end
end
```

Reply:
Totally agree.  But, yes - one for the future!